src/display/annotation_layer.js

```diff
@@ -730,6 +730,7 @@ class AnnotationElement {
         source: this,
         mode,
         editId,
+        mustEnterInEditMode: true,
```

Comment:
I'm probably missing something here, but `mustEnterInEditMode` (which I think is responsible for switching the mode on click) only seems to be set to `true` here, but this is the double-click handler so I don't really understand how a single click also achieves this behavior? The integration test shows that it works, I just don't really understand how it works exactly.

Reply:
This handler is called when dbl-clicking on an existing annotation when being in non-edit mode.
The single click handler is called when we're already in edit mode.